locale/fi/LC_MESSAGES/django.po

```diff
@@ -3635,3 +3643,6 @@ msgstr "Unohda automaattinen uudelleenohjaus."
 #: userprofile/views.py:80
 msgid "No privacy notice. Please notify administration!"
 msgstr "Ei tietosuojailmoitusta. Ota yhteys ylläpitäjään!"
+
+#~ msgid "Open chapter"
```

Comment:
The old string "open chapter" should be removed if it is not used anywhere. I guess the makemessages command commented it out automatically.